addons/hr_timesheet/views/project_project_views.xml

```diff
@@ -89,6 +89,7 @@
 
         <record id="project.open_view_project_all" model="ir.actions.act_window">
             <field name="domain">[('is_internal_project', '=', False)]</field>
+            <field name="target">current</field>
```

Comment:
I am not sure if this would solve the problem reported by Alexandra entirely, as this change is done in hr_timesheet and the problem was in the dashboard app. 
What would happen if the user has the project app and the dashboard app installed but not the timesheets app ?

Reply:
Hello

Done changes

Users with group_ids group_hr_timesheet_approver can view that's why changed in hr_timesheet: https://github.com/odoo/odoo/blob/16.0/addons/spreadsheet_dashboard_hr_timesheet/data/dashboards.xml#LL8C71-L8C98

But if don't update the project action it will create problems in the breadcrumb wherever we use that
action so removed main from project action because the target has a default value `current`

Thanks